jormungandr-integration-tests/src/common/jcli_wrapper/mod.rs

```diff
@@ -388,6 +354,43 @@ pub fn assert_transaction_log_shows_rejected(host: &str, expected_msg: &str) {
     }
 }
 
+pub fn assert_all_transactions_in_block(transactions_messages: &Vec<String>, host: &str) {
+    for transactions_message in transactions_messages.iter() {
+        self::assert_post_transaction(&transactions_message, &host);
```

Comment:
The use of `self::` here is a bit weird. Usually if there is a naming conflict, the imported freestanding functions are disambiguated with the module name, which is a good practice in general.